main.go

```diff
@@ -128,10 +130,20 @@ func proxy(w http.ResponseWriter, r *http.Request) {
 	setStrHeader(w, "Cache-Control", attr.CacheControl)
 	setStrHeader(w, "Content-Encoding", attrC)
 	setStrHeader(w, "Content-Disposition", attr.ContentDisposition)
-	setIntHeader(w, "Content-Length", attr.Size)
+	if gzipAcceptable {
+		setIntHeader(w, "Content-Length", attr.Size)
+	}
 	io.Copy(w, objr)
 }
 
+func clientAcceptsGzip(r *http.Request) bool {
+	acceptHeader := r.Header.Get("Accept-Encoding")
+	if strings.Contains(acceptHeader, "gzip") {
```

Comment:
I believe this can just be `return strings.Contains(...)`

Reply:
You are absolutely correct. Tested it and works just peachy!